src/nc_stats.c

```diff
@@ -827,7 +827,7 @@ stats_listen(struct stats *st)
         log_error("socket failed: %s", strerror(errno));
         return NC_ERROR;
     }
-
+    nc_set_reuseport(st->sd);
```

Comment:
Looks good, SO_REUSEPORT would definitely help with errors I've encountered during startup when stats port is listening on all interfaces.